pvlib/test/test_celltemp.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Mar 29 10:07:25 2019
+
+@author: cwhanse
+"""
+import pandas as pd
+
+import pytest
+from pandas.util.testing import assert_frame_equal
+from numpy.testing import assert_allclose
+
+from pvlib import celltemp, pvsystem
+from pvlib._deprecation import pvlibDeprecationWarning
+
+from conftest import fail_on_pvlib_version
+
+
+def test_sapm_celltemp():
+    default = celltemp.sapm(900, 20, 5)
+    assert_allclose(default['temp_cell'], 43.509, 3)
+    assert_allclose(default['temp_module'], 40.809, 3)
+    assert_frame_equal(default, celltemp.sapm(900, 20, 5, [-3.47, -.0594, 3]))
+
+
+def test_sapm_celltemp_dict_like():
+    default = celltemp.sapm(900, 20, 5)
+    assert_allclose(default['temp_cell'], 43.509, 3)
+    assert_allclose(default['temp_module'], 40.809, 3)
+    model = {'a': -3.47, 'b': -.0594, 'deltaT': 3}
+    assert_frame_equal(default, celltemp.sapm(900, 20, 5, model))
+    model = pd.Series(model)
+    assert_frame_equal(default, celltemp.sapm(900, 20, 5, model))
+
+
+def test_sapm_celltemp_with_index():
+    times = pd.date_range(start='2015-01-01', end='2015-01-02', freq='12H')
+    temps = pd.Series([0, 10, 5], index=times)
+    irrads = pd.Series([0, 500, 0], index=times)
+    winds = pd.Series([10, 5, 0], index=times)
+
+    pvtemps = celltemp.sapm(irrads, temps, winds)
+
+    expected = pd.DataFrame({'temp_cell': [0., 23.06066166, 5.],
+                             'temp_module': [0., 21.56066166, 5.]},
+                            index=times)
+
+    assert_frame_equal(expected, pvtemps)
+
+
+def test_PVSystem_sapm_celltemp(mocker):
+    racking_model = 'roof_mount_cell_glassback'
+
+    system = pvsystem.PVSystem(racking_model=racking_model)
+    mocker.spy(celltemp, 'sapm')
+    temps = 25
+    irrads = 1000
+    winds = 1
+    out = system.sapm_celltemp(irrads, temps, winds)
```

Comment:
call the new function, not the deprecated function. Unless explicitly testing for deprecation warnings, we should no longer call the old functions.

Reply:
same comment in the tests below

Reply:
Line 59 and Line 120 are calling the wrapper to test the PVSystem method - is this not correct?

Reply:
my mistake 59 and 120. Comment applies to lines 72, 76, 83, 90, 101.